unit/testing-utils/require_symbol.cpp

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************\
+
+ Module: Unit test utilities
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+
+#include "require_symbol.h"
+#include "catch.hpp"
+
+const symbolt &require_symbol::require_symbol(
```

Comment:
There is a `require_type::require_symbol` utility already. Consider using it here and given that your function checks for the existence too maybe the name of the function can reflect it? Something like `require_symbol_exists`.

Reply:
I don't have a `typet` just an identifier I want to look up so can't make use of that. However your name is much better so will use that.